hieradata/integration.yaml

```diff
@@ -41,6 +41,8 @@ govuk::deploy::setup::ssh_keys:
 govuk_cdnlogs::monitoring_enabled: false
 
 govuk_crawler::seed_enable: true
+govuk_crawler::targets:
+  - 's3://govuk-mirror-integration'
```

Comment:
We currently only [enable syncs in Production](https://github.com/alphagov/govuk-puppet/blob/master/hieradata/production.yaml#L165). I think it's worth testing this in Integration just to check the sync and bucket serving the site works, but we won't be able to test the full fastly failover, and it can be removed later.